include/deal.II/dofs/dof_accessor.h

```diff
@@ -1945,6 +1945,55 @@ class DoFCellAccessor : public DoFAccessor<DoFHandlerType::dimension,
   void
   update_cell_dof_indices_cache() const;
 
+  /**
+   * @name Dealing with refinement indicators
+   */
+  /**
+   * @{
+   */
+
+  /**
+   * Return whether the p-refinement flag is set or not.
```

Comment:
here and in the following functions, can you add a comment that this can only be used if the DoFHandler in use is of type `hp::DoFHandler`.